vrk-kpa/suomidigi
# SUOPA-665 Author layouts

Check https://github.com/vrk-kpa/suomidigi/pull/106 before this one.

To test: 
- `make fresh`
- **Phase 1** Go to some article. _F.e. https://suomidigi.docker.sh/artikkelit/kuvantamismenetelmien-kehittyminen-tuo-uusia-ulottuvuuksia-metsanhoitoon_
- Edit the article and check that the article has a writer.  If so, make a mental note of the writers name and click on the link underneath writer selection tool which states "Complete the author information. Edit authors."
   - Search for the writer and edit the term
   - Fill in all the fields and click save
   - You should end up on the same article you left from, so save the article and view it. There should be an author on the top of the article as well as in the bottom of the article. The bottom one is themed with all the information
- **Phase 2** Go and edit the same author (taxonomy term) as in phase 1.
   - Click on the Yes tab and select a profile from the list and save the term.
   - Go and refresh the article (the same one from phase 1) and check that the author in the bottom of the page has changed to the profile you selected
- **Phase 3** Go to your profile https://suomidigi.docker.sh/user and edit it.
  - Fill in all the details on your profile and save your profile.
  - Go to the same article as in Phase 1 and 2. Edit the Article and type in your own name to the author field and hit enter. Don't remove the other author.
   - Save the article and go to edit the authors https://suomidigi.docker.sh/admin/structure/taxonomy/manage/author/overview 
    - Edit your name (author) and link the your user profile to your author term.
    - Save the term and head back to the article.
- Now you should see two authors in the top and in the bottom of the page.
- Also the Author -label should be plural now. You can test it by removing the other author.
